documentation/source/building.rst

```diff
@@ -222,6 +222,27 @@ GPI
         and loading from libraries that `aren't` prefixed with "lib".
         Paths `should not` contain commas.
 
+PyGPI
+-----
+
+.. envvar:: COCOTB_ENTRY_POINT
+
+    The Python module and callable that starts up the Python cosimulation environment.
+    This defaults to :value:`cocotb:_initialise_testbench`, which is the cocotb standard entry point.
```

Comment:
The `:value:` isn't rendering on the docs.

Reply:
Yeah it should be `:data:`. I keep getting that one wrong.

Reply:
```suggestion
    This defaults to :data:`cocotb:_initialise_testbench`, which is the cocotb standard entry point.
```